test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -301,5 +319,14 @@ public static IEnumerable<object[]> MessageSizesData
                 new object[] { TransportType.ServerSentEvents },
                 new object[] { TransportType.LongPolling }
             };
+
+        public static IEnumerable<object[]> TransportTypesXTransferModes()
```

Comment:
The thing is that this is not *And* - this is Cartesian product. We also have [one more instance](https://github.com/aspnet/SignalR/blob/3a1d4c5dd66cdcccbc61dc8e530784c8b79b9b1b/test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs#L220) of this so if we want to change the convention it should be in both places.